### PR TITLE
Changes to make the ELB handlers more org-agnostic

### PR DESCRIPTION
**Pull request by fritzdj**
Changes to make the ELB handlers more org-agnostic + some sonar violation fixes

### CODE REVIEW

File: src/main/java/com/libertymutualgroup/herman/aws/cft/CftPush.java [hunk 1]
```diff
@@ -75,6 +74,7 @@
     private static final String MAVEN_VERS = "maven.version";
     private static final int RANDOM_PASSWORD_LENGTH = 20;
     private static final int POLLING_INTERVAL_MS = 10000;
+    private static final List<String> CFT_FILE_NAMES = Arrays.asList("cft.template", "cft.yml", "cft.json");
```
Comment (fritzdj) [inline src/main/java/com/libertymutualgroup/herman/aws/cft/CftPush.java]: Unrelated change - Adding support for different filenames here too. In the future I'll try to split out PRs in a coherent way.

File: src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerHandler.java [hunk 2]
```diff
@@ -124,21 +120,36 @@ public LoadBalancer createLoadBalancer(EcsClusterMetadata clusterMetadata, EcsPu
             elbClient.createLoadBalancer(createLoadBalancerRequest);
         } catch (DuplicateLoadBalancerNameException e) {
             LOGGER.debug("Error creating ELB: " + appName, e);
+
+            LoadBalancerDescription loadBalancerDescription = elbClient.describeLoadBalancers(
+                    new DescribeLoadBalancersRequest().withLoadBalancerNames(appName))
+                .getLoadBalancerDescriptions().get(0);
+            if (!elbScheme.equals(loadBalancerDescription.getScheme())) {
+                throw new AwsExecException(String.format("Expected scheme (%s) and actual scheme (%s) do not match. "
+                    + "Load balancer %s must be re-created.", elbScheme, loadBalancerDescription.getScheme(), appName));
```
Comment (fritzdj) [inline src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerHandler.java]: I could find a way to convert from one scheme to another for ELBs, so forcing folks to re-create if scheme is not what's expected. Same was added to the ELBv2 handler.

File: src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerHandler.java [hunk 3]
```diff
@@ -84,37 +83,34 @@ public LoadBalancer createLoadBalancer(EcsClusterMetadata clusterMetadata, EcsPu
...
         } else {
             elbScheme = "internal";
             elbSubnets = clusterMetadata.getElbSubnets();
         }
 
-        // Handle akamai
         List<String> elbSecurityGroups = new ArrayList<>();
-        if (isInternetFacingUrlScheme && HTTPS.equals(protocol)) {
-            elbSecurityGroups.addAll(clusterMetadata.getAkamaiSecurityGroup());
+        if ("internet-facing".equals(elbScheme) && HTTPS.equals(protocol)) {
```
Comment (c1phr) [inline src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerHandler.java]: Minor but I wonder if checking for "internet-facing" should go into a helper method?

Comment (fritzdj) [reply src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerHandler.java]: Good thought, I'll take a look at refactoring this later

**Pull request merged**